src/Context/wishlistContext.js

```diff
@@ -0,0 +1,37 @@
+import { createContext, useContext, useReducer } from "react";
+
+const WishlistContext = createContext();
+
+const wishFunction = (wishState, action) =>{
+    switch(action.type){
```

Comment:
same as cart, separate the reducer function

Reply:
Okay